main.js

```diff
@@ -0,0 +1,54 @@
+$(document).ready(function () {
+  const apiKey = '9a8359cd2576df91c6c91e74bb622c7a';
+  const baseUrl = 'http://api.openweathermap.org/data/2.5';
+  const weatherInfo = $('#weatherInfo');
+  const forecastInfo = $('#forecastInfo');
+  let city = '';
+
+  $('#searchBtn').click(function () {
+      city = $('#cityInput').val().trim();
+      if (city === '') {
+          alert('Please enter a city name');
+          return;
+      }
+
+      fetchCurrentWeather();
+      fetchForecast();
+  });
+
+  function fetchCurrentWeather() {
+    const currentWeatherUrl = `${baseUrl}/weather?q=${city}&appid=${apiKey}&units=imperial`;
+    fetch(currentWeatherUrl)
+        .then(response => response.json())
+        .then(data => {
+            const { name, main, weather } = data;
+            const currentWeatherHtml = `
+                <h2>Current Weather in ${name}</h2>
+                <p>Temperature: ${main.temp}°F</p>
+                <p>Conditions: ${weather[0].description}</p>
+                <img src="http://openweathermap.org/img/wn/${weather[0].icon}.png">
+            `;
+            weatherInfo.html(currentWeatherHtml);
```

Comment:
separating responsibilities and creating functions can help with readability.
This is not really fetching the weather, this is already rendering something... (hint for function name)